domain/src/main/java/me/androidbox/domain/authentication/model/LoginModel.kt

```diff
@@ -0,0 +1,7 @@
+package me.androidbox.domain.authentication.model
+
+data class LoginModel(
```

Comment:
No need to add the word "Model" to the models
For stuff like DTO's and Entities, it makes sense as to know which is which. But for domain models, which are used across the whole app, it's not needed nor recommended

You could simply name it Login or something a bit more descriptive like UserInformation